include/OpenSpaceToolkit/Astrodynamics/Trajectory.hpp

```diff
@@ -42,6 +43,11 @@ using ostk::physics::unit::Derived;
 using ostk::astrodynamics::trajectory::Model;
 using ostk::astrodynamics::trajectory::State;
 
+namespace trajectory
+{
+class Orbit;
```

Comment:
forward declaring instead of importing here to avoid cyclical dependency? 

nit: Does this not work?
```suggestion
class trajectory::Orbit;
```

Reply:
It didn't work when I tried it as the namespace trajectory does not exist.